.travis.yml

```diff
@@ -6,20 +6,24 @@ go:
 services:
   - rabbitmq
 
+sudo: required
+
 matrix:
   fast_finish: true
 
-# Only build pushes to the master branch (and PRs)
```

Comment:
Yep, this was an accidental inclusion that I meant to revert before @rolandshoemaker merged. I'll be reverting it when I merge @Hainish's integration test branch.